library/src/main/java/com/novoda/downloadmanager/lib/DownloadScanner.java

```diff
@@ -58,12 +58,16 @@ public void exec(MediaScannerConnection conn) {
         }
     }
 
-    //    @GuardedBy("mConnection")
-    private HashMap<String, ScanRequest> mPending = new HashMap<String, ScanRequest>();
+    //    @GuardedBy("mediaScannerConnection")
+    private HashMap<String, ScanRequest> pendingRequests = new HashMap<>();
```

Comment:
Is there something that requires `HashMap` explicitly? If not, this should be typed against the interface - `Map`.